easybuild/easyconfigs/c/Cereal/Cereal-1.3.0.eb

```diff
@@ -0,0 +1,30 @@
+##
+# @Authors:: Maxime Boissonneault, Compute Canada
+# @date: October 15, 2020
+##
+
+easyblock = 'CMakeMake'
+
+name = 'Cereal'
+version = '1.3.0'
+
+homepage = 'https://uscilab.github.io/cereal/'
+description = """cereal is a header-only C++11 serialization library. cereal takes arbitrary data types and reversibly 
+turns them into different representations, such as compact binary encodings, XML, or JSON. cereal was designed to be
+fast, light-weight, and easy to extend - it has no external dependencies and can be easily bundled with other code or 
+used standalone."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/USCiLab/cereal/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['329ea3e3130b026c03a4acc50e168e7daff4e6e661bc6a7dfec0d77b570851d5']
+builddependencies = [('CMake', '3.12.1')]
```

Comment:
We've learned the hard way that depending on a CMake built with the system compiler is troublesome in the long run, see #8261, so maybe it's better to use a CMake built with `GCCcore` as a build dep here (top-level `toolchain` can stick to `SYSTEM`, like we do for `Eigen`).